manual_jb/content/analyses/non_structural_building_dmg.md

```diff
@@ -24,8 +25,7 @@ key name | type | name | description
 `result_name` <sup>*</sup> | `str` | Result name | Name of the result dataset.
 `hazard_type` | `str` | Hazard type | Hazard type (earthquake).
 `hazard_id` | `str` | Hazard id | ID of the hazard from the Hazard service.
-`fragility_key_as` | `str` | AS fragility | Acceleration-sensitive fragility key used in mapping dataset.
-`fragility_key_ds` | `str` | DS fragility | Drift-sensitive fragility key used in mapping dataset.
+`fragility_key` | `str` | AS fragility | Acceleration-sensitive fragility key used in mapping dataset.
```

Comment:
This should probably be something more generic like:
 Fragility key | Fragility key used in mapping dataset.